src/content/app/genome-browser/components/interstitial/BrowserInterstitial.tsx

```diff
@@ -72,8 +75,26 @@ const BrowserInterstitial = () => {
 const ExampleLinks = () => {
   const { genomeIdForUrl } = useGenomeBrowserIds();
   const focusObjects = useAppSelector(getExampleGenes);
+  const focusLocations = useAppSelector(getExampleLocations);
+
+  const geneLinks = focusObjects.map((exampleObject) => {
+    const parsedFocusObjectId = parseFocusObjectId(exampleObject.object_id);
+    const focusId = buildFocusIdForUrl(parsedFocusObjectId);
+    const path = urlFor.browser({
+      genomeId: genomeIdForUrl,
+      focus: focusId
+    });
+
+    return (
+      <div key={exampleObject.object_id}>
+        <Link to={path} replace>
+          Example {exampleObject.type}
+        </Link>
+      </div>
+    );
+  });
 
-  const links = focusObjects.map((exampleObject) => {
+  const locationLinks = focusLocations.map((exampleObject) => {
```

Comment:
What is different in how location links are processed as opposed to gene links? Why is this a separate function?

Reply:
They are almost the same. Removed the code for `locationLinks`.

✅